test/integration/kpatch-test

```diff
@@ -254,6 +258,7 @@ run_combined_test() {
 
 	for testprog in "${TEST_LIST[@]}"; do
 		[[ $testprog != *-LOADED.test ]] && continue
+		[[ $testprog =~ -SLOW ]] && continue
```

Comment:
I realize this is consistent with existing behavior, but why don't we include -SLOW patches in the combined test?  Should we revisit that decision?

Reply:
I think this was part of the original setup that Seth created.  But yeah, if we're going to build all the patches (and most kernel objects) anyway, I don't think the -SLOW patches will add much to the build time.

Reply:
Looking back, it was actually me :-) about four years ago.  The original intent was that 'combined' was supposed to be a quick sanity test, so it avoided the slow rebuilds.  But if we're going to be doing combined patches for CI, I think the -SLOW patches should be included in that.  So maybe we can just get rid of the -SLOW concept altogether.